src/styles/bloks/stopwatch.scss

```diff
@@ -0,0 +1,79 @@
+.stopwatch {
+  margin: auto;
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 700px;
+  height: 700px;
```

Comment:
follow this requirement:
![image](https://github.com/mate-academy/layout_stop-watch/assets/108538921/fdad0502-2c51-4050-bd2b-758fdbfd22ea)